src/ml_gym/frontend/dashboard/src/features/jobsStatus/jobsStatusSlice.tsx

```diff
@@ -0,0 +1,50 @@
+import { createSlice } from '@reduxjs/toolkit'
+import { JobStatusMessageCollectionType, JobStatusType, JobStatusRowType} from '../../app/datatypes'
```

Comment:
Codacy has a fix for the issue: [Replace `·JobStatusMessageCollectionType,·JobStatusType,·JobStatusRowType}·from·'../../app/datatypes'` with `⏎··JobStatusMessageCollectionType,⏎··JobStatusType,⏎··JobStatusRowType,⏎}·from·"../../app/datatypes";`](https://app.codacy.com/gh/mlgym/mlgym/pullRequest?prid=9966992)
```suggestion
import {
```